doc/guides/releases.md

```diff
@@ -598,6 +598,7 @@ $ ./tools/release.sh -i ~/.ssh/node_id_rsa
 
 `tools/release.sh` will perform the following actions when run:
 
+<details>
 **a.** Select a GPG key from your private keys. It will use a command similar
```

Comment:
```suggestion 

**a.** Select a GPG key from your private keys. It will use a command similar
```

(Since otherwise this breaks formatting on Github)